osmaxx/excerptexport/models/export.py

```diff
@@ -163,6 +163,10 @@ def is_status_final(self):
         return self.status in self.FINAL_STATUSES
 
     @property
+    def can_be_deleted(self):
+        return self.is_status_final or self.update_overdue
```

Comment:
I had this name first, but I disliked the PyCharm not knowing and underlining it every time; it even proposed `is_delete_able` as alternative ^^